src/blockchain/test_utils/contracts/prevblocks.tolk

```diff
@@ -11,41 +11,27 @@ fun blockIdSeqno(blockId: tuple): int {
     return blockId.get(2);
 }
 
-fun onInternalMessage(myBalance: int, msgValue: int, msgFull: cell, msgBody: slice) {
-    if (msgBody.isEnd()) {
-        return;
-    }
-
-    var cs = msgFull.beginParse();
-    val flags = cs.loadMessageFlags();
-    if (isMessageBounced(flags)) {
```

Comment:
Why we did remove the bounced handler from here?

Reply:
Tolk has new way of message handling. Bounced messages do not accepted by on Internal message handler
https://docs.ton.org/v3/documentation/smart-contracts/tolk/tolk-vs-func/in-detail#modern-oninternalmessage